pkg/registry/reference.go

```diff
@@ -76,3 +77,8 @@ func (r *reference) String() string {
 	}
 	return r.orasReference.String()
 }
+
+// IsOCI determines whether a URL is to be treated as an OCI URL
+func IsOCI(url string) bool {
```

Comment:
I'm curious why you put this here ?

Reply:
This file contains `newReference` which trims off `oci://` from references and parses out registry repository and tag.

Maybe you meant why did I put it at the bottom instead of the top of the file though.